test/functional/ui/messages_spec.lua

```diff
@@ -391,10 +392,52 @@ describe('ui/ext_messages', function()
       },
     })
 
+    feed('<CR>')
+    n.add_builddir_to_rtp()
+    feed(':help<CR>:tselect<CR>')
+    local tagfile = t.paths.test_build_dir .. '/runtime/doc/help.txt'
```

Comment:
This test fails locally for me. It seems that `build/` comes after `$VIMRUNTIME` in `'rtp'`, so it may find `runtime/doc/help.txt` instead of `build/runtime/doc/help.txt`.

Reply:
Addressed in #32864.